### ISSUE TITLE
Loss not decreasing in stage 1

### ISSUE THREAD

**Issue by Bismuth209** [guoqincode/AnimateAnyone-unofficial#21]
After training stage-1 for 30000 steps on TikTok dataset I'm getting the following loss curve and images from `validation_pipeline` is this correct?

![image](https://github.com/guoqincode/AnimateAnyone-unofficial/assets/153806859/6f2b708e-9f04-46d2-bb5c-b5808ae7677b)

**Comment by guoqincode** [comment]
I also encountered this problem during the initial training. Could you tell me your detailed training config?

**Comment by Bismuth209** [comment]
https://github.com/Bismuth209/AnimateAnyone-unofficial/blob/main/configs/training/my_train_stage_1.yaml

P.S. is there a way directly connect on some kind of chat with you (I'll edit this line out once you respond either way)

**Comment by guoqincode** [comment]
guoqin@stu.pku.edu.cn

**Comment by Bismuth209** [comment]
I have a couple of questions

1. Were you able to train stage 1 with decreasing loss?
2. In stage 1 training, I see that you are not loading motion modules. wouldn't it be beneficial to load them from animate-diff?

**Comment by guoqincode** [comment]
1. The loss is high at the beginning and becomes low after a few hundred iterations
2. The first stage does not need to train the motion module, just the image level pre-training

**Comment by Bismuth209** [comment]
How do you verify if the first stage has passed?

**Comment by guoqincode** [comment]
Test the results on the validation set.

**Comment by Bismuth209** [comment]
can you show sample results that you received? I have shared what I was getting at the top of the comments

**Comment by tdlhyj** [comment]
I use the latest training code, but the loss does not decrease
